pkg/config/configobserver/configobserver.go

```diff
@@ -50,6 +50,9 @@ func New(defaultConfig config.Controller, kubeClient kubernetes.Interface) *Cont
 
 func (c *Controller) Start(ctx context.Context) {
 	wait.Until(func() {
+		if err := c.retrieveToken(); err != nil {
+			klog.Warningf("Unable to retrieve initial token config: %v", err)
```

Comment:
s/initial//

Reply:
@smarterclayton should this propagate to operator status (unavailable/degraded=true ?)

Reply:
Done

Reply:
We'll get notified about that at the next attempt to actually use the pull secret.